### PR TITLE
Add info command

### PR DESCRIPTION
**Pull request by bronzdoc**
# Description:

Adds back info command without the failing spec

I will abide by the [code of conduct](https://github.com/gemstash/gemstash/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by rsanheim** [comment]
This is pretty old, but wondering if it could get merged? I'm having a difficult time figuring out the current state of gemstash with some different configs I'm trying - this would be useful.

**Review by indirect — APPROVED** [review]
Going to defer to @olleolleolle on the actual merge, but I'm 👍🏻 on this.

File: spec/gemstash/cli/info_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    Gemstash::Env.current = @test_env
+  end
+
+  let(:cli_options) do
+    {
+      config_file: File.join(TEST_BASE_PATH, "info_spec_config.yml")
+    }
+  end
+
+  it "Prints the current current Gemstash configuration" do
```
Comment (olleolleolle) [inline spec/gemstash/cli/info_spec.rb:49]:
```suggestion
  it "prints the current current Gemstash configuration" do
```

Minor: Perhaps the `it` starts on a lower-case letter.

**Review by olleolleolle — APPROVED** [review]
So nice, I am in favor and thanks, @bronzdoc!

File: spec/gemstash/cli/info_spec.rb (hunk 1 above)
Comment (olleolleolle) [reply spec/gemstash/cli/info_spec.rb:49]: ("We can fix it in post" as the film-makers say.)

**Pull request merged**